examples/controls_example/public/app/react_control_example.tsx

```diff
@@ -263,6 +270,28 @@ export const ReactControlExample = ({
             }}
           />
         </EuiFlexItem>
+        <EuiFlexItem grow={false}>
```

Comment:
I don't think this is needed. You can just run the page without sample web logs to see errors in controls